packages/medusa/src/api/store/customers/query-config.ts

```diff
@@ -38,6 +38,10 @@ export const defaultStoreCustomerAddressFields = [
 
 export const retrieveAddressTransformQueryConfig = {
   defaults: defaultStoreCustomerAddressFields,
+  allowed: [
```

Comment:
todo: Can you also add `allowed` to `retrieveTransformQueryConfig` in this file? 🙏

Reply:
And remove `orders` here. This is for customer addresses

Reply:
done ✅